Source/ItemDatabase.lua

```diff
@@ -7,8 +7,8 @@ local FuzzyMatcher = require 'Utility.FuzzyMatcher'
 -- Consts
 local const = util.ReadOnly({
   -- Find highest ID @ https://classic.wowhead.com/items?filter=151;2;24283
-  disjunctItemIds = {172070},
-  highestItemId = 24283,
+  disjunctItemIds = {172070, 12270, 122284, 180089},
```

Comment:
The `12270` ID is already covered by the first range in `itemIdRanges`. Is the ID redundant or a typo?

Reply:
Most likely a typo, I'll go back and have a second pass :D